R/emuFit.R

```diff
@@ -400,7 +429,8 @@ and the corresponding gradient function to constraint_grad_fn.")
                                          n = n,
                                          p = p, 
                                          I_inv=I_inv,
-                                         Dy = just_wald_things$Dy)*(n/(n - 1))
+                                         Dy = just_wald_things$Dy,
+                                         cluster = cluster)*score_adj
```

Comment:
Can you please confirm that score test results will be unchanged from previous versions? I expect that moving from multiplying Dy by this factor to multiplying the score statistic by this factor shouldn't change anything, but would love to confirm as this seems very important.

Reply:
So I initially read the (pre-PR) code the same way -- the syntax is not particularly clear, but as things were, we were multiplying the whole score statistic by (n/(n - 1)). (The close paren following Dy is the end of a call to get_score_stat.) So there should be no difference whatsoever!